src/storage/query/GetNeighborsProcessor.cpp

```diff
@@ -153,22 +176,42 @@ folly::Future<std::pair<nebula::cpp2::ErrorCode, PartitionID>> GetNeighborsProce
     RuntimeContext* context,
     StorageExpressionContext* expCtx,
     nebula::DataSet* result,
+    cpp2::GetNeighborsRequest& req,
```

Comment:
It seems the req is not used in this function, would you remove it? We can get rid of `const_cast` as well

Reply:
The variables transferred from graphd are stored in req, and the following FilterNode are useful

Reply:
Sorry I saw input, I deleted him

Reply:
done